### ISSUE TITLE
Default argument values

### ISSUE THREAD

**Issue by DeflatedPickle** [sflynlang/sflynlang-compiler#20]
Sometimes functions are written that need to be called multiple times after each other, with the same arguments, or have an argument that is often the same apart from a few cases

Whilst a wrapper function could be written, that would require all arguments apart from the optional ones to be written, which is quite verbose

So, I suggest a simple `=` to denote what the value would be if none were passed
```
class User(name: string, will: boolean = false) {}
func welcome(name: string, positive: boolean = true) {}
```

**Comment by danielsolartech** [comment]
Of course, you can look an example of that here: https://github.com/sflynlang/sflynlang-compiler/blob/main/examples/functions/pow_default_value.sf

And, I'm thinking in add the possibility of set the name of the argument to skip the other default arguments, like this:
```sf
func welcome(name: string = "Sflyn", positive: boolean = true) {
    print(`${name} - ${positive.toString()}`);
}

welcome("Sflynlang");
// Output: Sflynlang - true

welcome(positive = false);
// Output: Sflyn - false

welcome("Sflynlang", false);
// Output: Sflynlang - false

welcome(positive = false, name = "Sflynlang");
// Output: Sflynlang - false
```

**Comment by DeflatedPickle** [comment]
Ah, I must have missed that! But named arguments would be a great addition
